lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -278,18 +278,24 @@ def self.run_dsym_upload(params)
 
       # run whole upload process for release
       def self.run_release_upload(params)
+        values = params.values
         api_token = params[:api_token]
         owner_name = params[:owner_name]
         app_name = params[:app_name]
         group = params[:group]
         release_notes = params[:release_notes]
+        release_notes_clipping = params[:release_notes_clipping]
+        release_notes_link = params[:release_notes_link]
 
         if release_notes.length >= Constants::MAX_RELEASE_NOTES_LENGTH
-          clip = UI.confirm("The release notes are limited to #{Constants::MAX_RELEASE_NOTES_LENGTH} characters, proceeding will clip them. Proceed anyway?")
-          UI.abort_with_message!("Upload aborted, please edit your release notes") unless clip
-          release_notes_link = UI.input("Provide a link for additional release notes, leave blank to skip")
+          unless release_notes_clipping
+            clip = UI.confirm("The release notes are limited to #{Constants::MAX_RELEASE_NOTES_LENGTH} characters, proceeding will clip them. Proceed anyway?")
```

Comment:
🔧 The name for this could be slightly more descriptive, maybe `should_clip`? Not sure if there's a Ruby naming convention for this 🙂

Reply:
Sounds good, I agree